src/logger/console.logger.ts

```diff
@@ -0,0 +1,38 @@
+import { ExtendedLogger } from "./logger.types"
+
+// NOTE: See [here](https://talyian.github.io/ansicolors/)
+// for an overview of the color codes.
+// Do note that this functions by having the form:
+// `{color code}{message}{reset code}`
+// where %s is where the message gets injected
+const COLORS = {
+  FOREGROUND: {
+    RED: "\x1b[31m%s\x1b[0m",
+    GREEN: "\x1b[32m%s\x1b[0m",
+    YELLOW: "\x1b[33m%s\x1b[0m",
+    BLUE: "\x1b[34m%s\x1b[0m",
+  },
+  BACKGROUND: {
+    RED: "\x1b[41m%s\x1b[0m",
```

Comment:
why is this a background.red?

Reply:
like we agreed above, fatal should be background red, no? we already use foreground red for `error` and this is more severe, so this is bg red.